src/nvim/ex_cmds.c

```diff
@@ -1892,7 +1892,7 @@ int do_write(exarg_T *eap)
     if (eap->cmdidx == CMD_saveas) {
       if (retval == OK) {
         curbuf->b_p_ro = FALSE;
```

Comment:
Missed a FALSE

Reply:
Opted to NOLINT to avoid clint chain reaction.